src/main/java/frc/robot/hardware/phoenix6/BusChain.java

```diff
@@ -0,0 +1,105 @@
+package frc.robot.hardware.phoenix6;
+
+import com.ctre.phoenix6.CANBus;
+import frc.utils.alerts.Alert;
+import frc.utils.alerts.AlertManager;
+import frc.utils.alerts.PeriodicAlert;
+import org.littletonrobotics.junction.Logger;
+import com.ctre.phoenix6.CANBus.CANBusStatus;
+
+public enum BusChain {
+
+	ROBORIO("rio"),
+	CANIVORE("CANivore");
+
+	private static final double PERMITTED_CAN_UTILIZATION_DECIMAL_VALUE = 0.6;
```

Comment:
Why is this not 0?

Reply:
the can utilization is between 0 - 1, the utilization we consider as not problematic is less than 0.6

Reply:
Change this to 0.0 so we will know to calibrate this when creating the robot during the season

Reply:
But for now it’s same value for all robots.
Template can have plug and play code if it’s generic to each robot